main/modes/tunernome/tunernome.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * tunernome.h
+ *
+ *  Created on: September 17th, 2020
+ *      Author: bryce
+ */
+
+#ifndef _TUNERNOME_H_
+#define _TUNERNOME_H_
+
+#include "swadge2024.h"
+
+extern swadgeMode_t modeTunernome;
+
+extern const song_t metronome_primary;
+extern const song_t metronome_secondary;
```

Comment:
These shouldn't be declared `extern` unless they're used elsewhere

Reply:
They were used in the jukebox SFX section, which is still planned to be implemented

Reply:
They're mostly so that we have pre-made arrows ready for any size required. An example would be when a new font is made, and an arrow of exact matching height is needed.

It makes more sense to have these in a non-building directory ready for quick insertion; not all taking up space in the FW partitions. Any preference for where to put something like this?

Reply:
This is how we end up with no common design language despite no intended difference, but if you think this is best, I'll do it

Reply:
I'm leaving the 8px arrow in so we have one standard arrow of the same design for each font.